panther_gazebo/launch/simulate_robot.launch.py

```diff
@@ -185,6 +185,19 @@ def generate_launch_description():
         emulate_tty=True,
     )
 
+    docking_launch = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            PathJoinSubstitution(
+                [
+                    FindPackageShare("panther_docking"),
+                    "launch",
+                    "docking.launch.py",
+                ]
+            ),
+        ),
+        launch_arguments={"namespace": namespace, "use_sim": "True"}.items(),
```

Comment:
Use docking condition should be used here
```suggestion
        condition=IfCondition(use_docking),
        launch_arguments={"namespace": namespace, "use_sim": "True"}.items(),
```

Reply:
This may seem strange, but we already have it done in other packages as well (on latest `ros2-lynx-devel`). Since the arguments are passed to the parent launch file, there is no need to specify this parameter in bringup.launch.py ​​and simulation.launch.py, which improves the readability of these two most important files.

We can discuss about this, but not in this task: